src/PlaywrightSharp/Contracts/IFileChooser.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace PlaywrightSharp
+{
+    public partial interface IFileChooser
+    {
+        /// <inheritdoc cref="SetFilesAsync(System.Collections.Generic.IEnumerable{string}, bool?, float?)"/>
+        Task SetFilesAsync(string files, bool? noWaitAfter = default, float? timeout = default);
```

Comment:
With this we support the `[path]` overload defined here https://github.com/microsoft/playwright/blame/master/docs/src/api/params.md#L70